SpatialGDK/Documentation/known-issues.md

```diff
@@ -12,8 +12,8 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | Stably-named replicated actors cannot be referred to by their path | 2018-08-10 | [UNR-473](https://improbableio.atlassian.net/projects/UNR/issues/UNR-473) | None |
 | Server travel does not work in PIE. | 2018-10-24 | | Server travel work must be done with external or managed workers |
 | Server travel does not work in multi-worker configurations.  | 2018-10-24 | [UNR-678](https://improbableio.atlassian.net/projects/UNR/issues/UNR-678)
-| We don't support the `ReplicateYes` policy on GameplayAbilities. | 2018-10-24 | [UNR-675](https://improbableio.atlassian.net/projects/UNR/issues/UNR-675) | Don't use replicated GameplayAbilities. If they need access to replicated data, store it on the AbilityComponent itself. |
-| We don't support NetDeltaSerialize or Fast TArray Replication.  | 2018-10-24 |  | Use default serialization |
+| `ReplicateYes` policy on GameplayAbilities not supported. | 2018-10-24 | [UNR-675](https://improbableio.atlassian.net/projects/UNR/issues/UNR-675) | Don't use replicated GameplayAbilities. If they need access to replicated data, store it on the AbilityComponent itself. |
```

Comment:
😧 why the change from active -> passive? (Not gonna fight for active at this point; just curious - feel free to ignore!)

Reply:
Dunno, "we don't support" or "GDK does not support" did not sound right. Not fussed either way but let's get this in for now :).